app/cafe.py

```diff
@@ -0,0 +1,24 @@
+import datetime
+from app.errors import (NotVaccinatedError, OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError(f"{visitor.get(
+                "name", "Unknown visitor")} is not vaccinated")
+
+        expiration_date = visitor["vaccine"].get("expiration_date")
+        if expiration_date < datetime.date.today():

```

Comment:
Ensure that `expiration_date` is not `None` before comparing it with `datetime.date.today()`. You might want to add a check to confirm that `expiration_date` is a valid date object to avoid potential errors.